src/wasm-lib/kcl/src/lib.rs

```diff
@@ -171,6 +185,7 @@ impl Program {
     pub fn change_meta_settings(&mut self, settings: crate::MetaSettings) -> Result<Self, KclError> {
```

Comment:
This is a weird method because it looks like it mutates the `self` passed in and returns a clone of itself? I'm a bit averse to cloning the program text (and the AST?) here, so could it take `self` by value or just mutate `self` rather than cloning?

Reply:
ah okay wasnt part of this pr but i can fix

Reply:
done!